be/src/storage/rowset/column_reader.cpp

```diff
@@ -604,14 +604,14 @@ StatusOr<std::vector<ZoneMapDetail>> ColumnReader::get_raw_zone_map(const IndexR
 
     LogicalType type = _encoding_info->type();
     int32_t num_pages = _zonemap_index->num_pages();
-    std::vector<ZoneMapDetail> result(num_pages);
+    std::vector<ZoneMapDetail> result;
```

Comment:
shall be marked as bugfix?

Reply:
it would not affect the correctness, we can ignore it